cg/meta/invoice.py

```diff
@@ -16,12 +16,12 @@ def __init__(self, db: Store, lims_api: lims.LimsAPI, invoice_obj: models.Invoic
         self.customer_obj  = invoice_obj.customer
 
     def prepare_contact_info(self, costcenter):
-        msg = f'Could not open/generate invoice. Contact information missing in database for customer {self.customer_obj.internal_id}. See log files.'
+        msg = f'Could not open/generate invoice. Contact information missing in Customer/User table for customer {self.customer_obj.internal_id}. See log files.'
```

Comment:
Why did you remove the information on which customer the problem arose? I guess that is generally helpful information?

Reply:
Didn't remove that. Just specifyed wich tables could be missing data. Customer id is still there!